scripts/load_data_into_db.py

```diff
@@ -0,0 +1,99 @@
+import getpass
+import pandas as pd
+import psycopg2
+import sys
+
+def create_table(hostname, dbname, user):
```

Comment:
This is really similar to the code inside of setup_db.py. Can we move this to the `create_tables` function in setup_db.py?